admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -1,58 +1,78 @@
-import { Breadcrumb, BreadcrumbItem, Button, Modal, ModalVariant, PageSection, Spinner, Title, Toolbar,
+import { Breadcrumb, BreadcrumbItem, Button, Modal, ModalVariant, PageSection,     Skeleton, Spinner, Title, Toolbar,
     ToolbarContent, ToolbarItem } from '@patternfly/react-core';
 import {
     ActionGroup,
     Form,
-    FormGroup,
-    TextArea,
+    FormGroup, TextArea,
     TextInput  } from '@patternfly/react-core';
-import { TableComposable, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+import { PencilAltIcon } from '@patternfly/react-icons';
+import {
+    TableComposable,
+    Tbody,
+    Td,  Th,   Thead,
+    Tr } from '@patternfly/react-table';
 import * as React from 'react';
-import { useState } from 'react';
 import { useParams } from 'react-router';
+import { style } from 'typestyle';
 
 import { useCreateEventType } from '../services/CreateEventTypes';
 import { useApplicationTypes } from '../services/GetApplication';
-import { useBundles } from '../services/GetBundles';
 import { useEventTypes } from '../services/GetEventTypes';
+import { EventType } from '../types/Notifications';
 
 type ApplicationPageParams = {
     applicationId: string;
 }
 
+const skeletonClassName = style({
+    width: 200
+});
+
 export const ApplicationPage: React.FunctionComponent = () => {
     const { applicationId } = useParams<ApplicationPageParams>();
     const eventTypesQuery = useEventTypes(applicationId);
     const applicationTypesQuery = useApplicationTypes(applicationId);
-    const getBundle = useBundles();
     const columns = [ 'Event Type', 'Name', 'Description', 'Event Type Id' ];
 
     const newEvent = useCreateEventType();
-    const [ id ] = React.useState<string | undefined>();
-    const [ displayName, setDisplayName ] = React.useState<string | undefined>();
-    const [ name, setName ] = React.useState<string | undefined>();
-    const [ description, setDescription ] = React.useState<string | undefined>();
+    const [ event, setEvent ] = React.useState<Partial<EventType>>({});
```

Comment:
Is this unrelated to this PR? Maybe waiting on the other PR or reverting 74d2ae037c2c42efb5d2a81b11cbf7e574ec5963 ?